lib/bindings/python/examples/hello_world/client.py

```diff
@@ -20,7 +20,7 @@
 from dynamo.runtime import DistributedRuntime, dynamo_worker
 
 
-@dynamo_worker()
+@dynamo_worker(static=True)
```

Comment:
Do you think there is a value in an hello world example demonstrating discoverability? I am not sure if one is already there.

Reply:
Yes, all the other examples do that. This is the only non-discoverable example.